templates/components/products/product-view.html

```diff
@@ -76,10 +76,39 @@ <h2 class="productView-brand"{{#if schema}} itemprop="brand" itemscope itemtype=
                     <dt class="productView-info-name">{{lang 'products.availability'}}</dt>
                     <dd class="productView-info-value">{{product.availability}}</dd>
                 {{/if}}
-                {{#if product.weight}}
+                {{#all product.weight theme_settings.show_product_weight}}
                     <dt class="productView-info-name">{{lang 'products.weight'}}</dt>
                     <dd class="productView-info-value" data-product-weight>{{product.weight}}</dd>
-                {{/if}}
+                {{/all}}
+                {{#all product.width product.height. product.depth theme_settings.show_product_dimensions}}
```

Comment:
there is an extra dit in `product.height.` 😛

Reply:
:woman_facepalming: ✔️